enterprise/api/v1/serializers.py

```diff
@@ -159,11 +162,15 @@ def validate_username(self, value):
         try:
             user = User.objects.get(username=value)
         except User.DoesNotExist:
+            error_message = '[Enterprise API] User with username {} does not exist.'.format(value)
```

Comment:
i would be more specific here; the username does not exist for creating an EnterpriseCourseEnrollment record

Reply:
Done